wercker.yml

```diff
@@ -5,11 +5,14 @@ services:
 build:
     steps:
         - virtualenv:
-            install_wheel: false
+            install_wheel: true
```

Comment:
I only briefly read up on it and took away it was kind of like a cache, impacting build speed. I had some trouble getting first build pass and it happened to be set to true on my first pass.  So I left it.